modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue

```diff
@@ -215,10 +247,8 @@ export default {
     form {
       z-index: $z-index-manager-toolbar;
```

Comment:
Every `z-index` rule was necessary with a background image and a filter. They can be removed now there is a white background.